Dockerfile

```diff
@@ -44,7 +44,7 @@ WORKDIR /jitsi-meet
 COPY package.json /jitsi-meet/package.json
 COPY package-lock.json /jitsi-meet/package-lock.json
 
-RUN npm i
+RUN npm i -g npm@7.5.4
```

Comment:
It's fine to update npm to 7.5.4, but you shouldn't remove the 'RUN npm i' line. That installs the dependencies of the package that's in the working directory. npm i -g <whatever> installs a package globally, in this case, a specific version of npm.